services/topology-engine/queue-engine/topologylistener/flow_utils.py

```diff
@@ -197,7 +197,7 @@ def merge_flow_relationship(flow, tx):
     # FIXME(surabujin): do we really want to keep this time representation?
     # FIXME(surabujin): format datetime as '1532609693'(don 't match with
     #                   format used in PCE/resource cache)
-    p['last_updated'] = str(int(time.time()))
+    p['last_updated'] = int(time.time())
```

Comment:
Can you handle "incorrect" from now values (created by code before this change)?